common/src-mapred/org/pentaho/hbase/mapred/PentahoTableInputFormat.java

```diff
@@ -106,6 +106,7 @@ public void configure(JobConf job) {
     
     // set our table record reader
     PentahoTableRecordReader rr = new PentahoTableRecordReader();
+    rr.setHTable( getHTable() );
```

Comment:
Is this necessary for the 5.4 version? or did it get backported from Master by mistake? Things are compiling file but I worry about its affect on existing shims besides cdh54.